examples/batch/main.cc

```diff
@@ -0,0 +1,101 @@
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+// Using an exporter that simply dumps span data to stdout.
+#include "opentelemetry/exporters/ostream/span_exporter.h"
+#include "opentelemetry/sdk/trace/batch_span_processor.h"
+
+#include <chrono>
+#include <thread>
+
+constexpr int kNumSpans = 10;
+
+namespace
+{
+
+void initTracer()
+{
+  /*                         SET UP EXPORTER                                    */
```

Comment:
Please remove comments that don't add much to the information that is already given by the code. I think 
```c++
  auto exporter = std::unique_ptr<sdktrace::SpanExporter>(
      new opentelemetry::exporter::trace::OStreamSpanExporter);
```
makes it clear that a new exporter is set up.